### PR TITLE
Bonnie 1087 excel export details

### PR DESCRIPTION
**Pull request by alexanderelliott121**
This pull request is intended to bring the excel export into alignment with user desires:
specifically, it properly populates the notes section and re-orders the name to be displayed last, first, so as to match the front end display.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1087
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced. N/A
- [X] Tests are included and test edge cases
- [X] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [X] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [X] JIRA test links: N/A
- [X] Justification for using JIRA tests: N/A
- [X] JIRA tests have been added to sprint: N/A

**Reviewer 1:**

Name: @losborne
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [X] JIRA tests have been run and pass N/A
- [X] You agree with the justification for use of JIRA tests or have provided input on why you disagree N/A

**Reviewer 2:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA tests have been run and pass
- [x] You agree with the justification for use of JIRA tests or have provided input on why you disagree

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -44,6 +44,9 @@ gem 'multi_json'
 # needed for parsing value sets (we need to use roo rather than rubyxl because the value sets are in xls rather than xlsx)
 gem 'roo'
 
+# needed for excel export test
+gem 'google_drive'
```
Comment (jbradl11) [inline Gemfile]: Is this needed? I ran the tests without this gem and all tests pass.

**Review by jbradl11 — CHANGES_REQUESTED** [review]
Looks like Race and Ethnicity are not populating in the spreadsheet.

**Review by lukoktonos — CHANGES_REQUESTED** [review]
The unit test needs to be extended to check for death date (with a filled in value if one exists--this works when I test manually), ethnicity and race (which don't work, as JB mentioned), and to ensure that red denotes failed test cases, if that is possible with the gem.

File: Gemfile (hunk 1 above)
Comment (alexanderelliott121) [reply Gemfile]: When I was running with the debugger, I had to include that in order to open files.

I'll try without it outside the debugger.

**Review by lukoktonos — APPROVED** [review]
Changes look good--race and ethnicity are populating correctly, and are tested.

**Review by jbradl11 — APPROVED** [review]
Code looks good. Tests pass. Notes / race / ethnicity all work now.

**Pull request merged**